crates/tests/src/integration/masp.rs

```diff
@@ -3208,7 +3269,7 @@ fn masp_fee_payment_with_custom_spending_key() -> Result<()> {
         )
     });
     assert!(captured.result.is_ok());
-    assert!(captured.contains("nam: 0"));
+    assert!(captured.contains("nam: 1000"));
```

Comment:
Could this be due to the `DenominatedAmount` thing I mentioned in the other file?